lib/ndr_import/non_tabular/column_mapping.rb

```diff
@@ -42,7 +43,7 @@ def capture_value(line)
             value = nil
           end
         end
-        value
+        value.nil? ? value : value.strip
```

Comment:
After discussion, using `value.try(:strip)` is slower than the ternary operator, and the mapper has a number of nested loops that could make this non-academic...

Reply:
(the answer would be benchmarking in-situ, rather than in isolation)